themes/sensei-course-theme/templates/quiz.html

```diff
@@ -1,40 +1,32 @@
-<!-- wp:sensei-lms/container {"className":"quiz"} -->
-<div class="quiz quiz-blocks">
-	<!-- wp:sensei-lms/container {"className":"sensei-course-theme__quiz", "tagName": "form"} -->
-	<form method="post" enctype="multipart/form-data" class="wp-block-sensei-lms-container sensei-course-theme__quiz">
-		<!-- wp:columns {"className":"sensei-course-theme__quiz__header sensei-course-theme__frame"} -->
-		<div class="wp-block-columns sensei-course-theme__quiz__header sensei-course-theme__frame">
-			<!-- wp:column {"className":"sensei-course-theme__quiz__header__left"} -->
-			<div class="wp-block-column sensei-course-theme__quiz__header__left">
-				<!-- wp:sensei-lms/quiz-back-to-lesson /-->
-				<!-- wp:post-title /-->
-			</div>
-			<!-- /wp:column -->
-			<!-- wp:column {"className":"sensei-course-theme__quiz__header__right"} -->
-			<div class="wp-block-column sensei-course-theme__quiz__header__right">
-				<!-- wp:sensei-lms/quiz-progress-counter /-->
-				<!-- wp:sensei-lms/quiz-progress-bar /-->
-			</div>
-			<!-- /wp:column -->
-		</div>
-		<!-- /wp:columns -->
-		<!-- wp:sensei-lms/container {"className":"sensei-course-theme__quiz__main-content"} -->
-		<div class="wp-block-sensei-lms-container sensei-course-theme__quiz__main-content">
-			<!-- wp:sensei-lms/course-theme-notices /-->
-			<!-- wp:sensei-lms/course-content /-->
-		</div>
-		<!-- /wp:sensei-lms/container -->
+<!-- wp:columns {"className":"sensei-course-theme__quiz__header sensei-course-theme__frame"} -->
+<div class="wp-block-columns sensei-course-theme__quiz__header sensei-course-theme__frame">
+	<!-- wp:column {"className":"sensei-course-theme__quiz__header__left"} -->
+	<div class="wp-block-column sensei-course-theme__quiz__header__left">
+		<!-- wp:sensei-lms/quiz-back-to-lesson /-->
+		<!-- wp:post-title /-->
+	</div>
+	<!-- /wp:column -->
+	<!-- wp:column {"className":"sensei-course-theme__quiz__header__right"} -->
+	<div class="wp-block-column sensei-course-theme__quiz__header__right">
+		<!-- wp:sensei-lms/quiz-progress-bar /-->
+	</div>
+	<!-- /wp:column -->
+</div>
+<!-- /wp:columns -->
+
+<!-- wp:group {"className":"sensei-course-theme__quiz__main-content", "tagName": "main"} -->
+<main class="wp-block-group sensei-course-theme__quiz__main-content">
+	<!-- wp:sensei-lms/course-theme-notices /-->
+	<!-- wp:sensei-lms/course-content /-->
```

Comment:
This block is with "Lesson Content" hard-coded. I think we can change it to something more generic to use in the quiz too.

Reply:
Updated in 04f0636